index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Design Teardown</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Spectral&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lora:wght@700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <nav class="navbar flex zero-border tier-2">
+      <p class="main-nav-logo tier-3">Main Logo</p>
+      <p class="nav-element tier-4">1</p>
+      <p class="nav-element tier-4">2</p>
+      <p class="nav-element tier-4">3</p>
+      <p class="nav-element tier-4">4</p>
+      <p class="nav-element tier-4">5</p>
+      <p class="nav-element tier-4">6</p>
+      <p class="nav-element tier-4">7</p>
+      <p class="search-nav tier-3">Search</p>
+    </nav>
+    <!-- <div class="test grid">
+      <div class="tier-1">One</div>
+      <div class="tier-2">Two</div>
+      <div class="tier-3">Three</div>
+      <div class="tier-4">Four</div>
+      <div class="tier-5">Five</div>
+    </div> -->
+    <section class="article-teaser tier-1">
+      <h2>Lorem ipsum dolor sit amet, consectetur.</h2>
+    </section>
+    <article class="first-article-grid grid tier-3">
+      <div>
+        <div class="fake-pic"></div>
+        <p>Lorem ipsum dolor.</p>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+      <div>
+        <div class="fake-pic"></div>
+        <p>Lorem ipsum dolor.</p>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+      <div>
+        <div class="fake-pic"></div>
+        <p>Lorem ipsum dolor.</p>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+      <div>
+        <div class="fake-pic"></div>
+        <p>Lorem ipsum dolor.</p>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+        <p>#lorem #lorem #lorem</p>
+      </div>
+    </article>
```

Comment:
- [OPTIONAL] You can use the different shades of gray for background of each item for better visualization.
![image](https://user-images.githubusercontent.com/60184961/91843976-50363700-ec5f-11ea-8445-52c5ebdae888.png)